udata/settings.py

```diff
@@ -494,6 +494,7 @@ class Testing(object):
         'title^2',
         'description',
     )
+    MONGODB_HOST_TEST = 'mongomock://localhost:27017/udata'
```

Comment:
Maybe update docs (`adapting-settings.md` and `testing-code.md`) with mongomock changes.